extension/docs/DEV.md

```diff
@@ -45,18 +45,16 @@ cp .env.example .env.production
 git submodule update --init --recursive
 ```
 
-2. Add the model files to be included in the build according to the instructions [here](../bergamot-translator/README.md).
+2. Acquire build artifacts bergamot-translator, according to the instructions [here](../bergamot-translator/README.md).
```

Comment:
I prefer this repo to be able to build bergamot-translator as part of its building step. We won't be able to pull pre-built artifacts when start incorporating this into threeherder

Reply:
Ok, sure, will revert when bergamot-translator builds reproducibly systems.